### DISCUSSION TITLE
Is open source still planned for WinUI3? Any timeline?

### DISCUSSION THREAD

**Discussion by fredemmott** [microsoft/microsoft-ui-xaml#7873] [Q&A]
https://microsoft.github.io/microsoft-ui-xaml/ implies that WinUI3 is open source, but it doesn't appear to be (yet?) - is there any news or timeline here?

In particular, if it were open source, I'd be taking a look at fixing https://github.com/microsoft/microsoft-ui-xaml/issues/7254 myself.

**Comment by gabbybilka** [comment]
@predavid as FYI
